src/httpserver.cpp

```diff
@@ -388,6 +388,12 @@ static bool HTTPBindAddresses(struct evhttp* http)
             if (i->first.empty() || (addr.has_value() && addr->IsBindAny())) {
                 LogPrintf("WARNING: the RPC server is not safe to expose to untrusted networks such as the public internet\n");
             }
+            // Set the no-delay option (disable Nagle's algorithm) on the TCP socket.
+            evutil_socket_t fd = evhttp_bound_socket_get_fd(bind_handle);
+            int nOne = 1;
```

Comment:
nit:  could use `n_one` to better align with style guidelines.

https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c
> Variable (including function arguments) and namespace names are all lowercase and may use _ to separate words (snake_case).

Reply:
Thanks! Fixed to `one` in c1bec82697.

Reply:
Ah, that's right (no hungarian). Thanks sipa